src/run_prettify.js

```diff
@@ -1662,7 +1665,7 @@ var IN_GLOBAL_SCOPE = false;
               'keywords': JSCRIPT_KEYWORDS,
               'cStyleComments': true,
               'regexLiterals': true
-            }), ['javascript', 'js']);
+            }), ['javascript', 'js', 'ts', 'typescript']);
```

Comment:
Note to self: we can think about `.jsx` down the road.

(Apparently Facebook reinvented E4X and is calling it React JS.)